packages/core-js/internals/symbol-constructor-detection.js

```diff
@@ -1,13 +1,16 @@
 /* eslint-disable es/no-symbol -- required for testing */
 var V8_VERSION = require('../internals/engine-v8-version');
 var fails = require('../internals/fails');
+var $String = String;
```

Comment:
In one moment, Terser could become smarter and "optimize" it even in this case. It's better to get `String` like here https://github.com/zloirock/core-js/blob/v3.30.1/packages/core-js/internals/regexp-sticky-helpers.js#L2-L5